app/models/site.rb

```diff
@@ -37,8 +37,12 @@ def user_fields
 
   CATEGORIES_CACHE_KEY = "site_categories"
```

Comment:
I don't see more references to CATEGORIES_CACHE_KEY, can we delete it?

Reply:
Oops good catch :+1: